.github/workflows/snap-testing.yml

```
@@ -60,4 +64,5 @@ jobs:
           name: ${{matrix.name}}
           channel: ${{matrix.channel}}
           platform_channel: ${{matrix.platform_channel}}
-          print_logs: ${{matrix.print_logs}}
+          full_config_test: ${{ github.event_name == 'schedule' && true || false }}
```

```suggestion
          full_config_test: true
```
Didn't you agree to enable by default?